pkg/config/config.go

```diff
@@ -146,14 +146,15 @@ func GetDefaultPowerModelURL(modelOutputType, energySource string) string {
 
 func logBoolConfigs() {
 	if klog.V(5).Enabled() {
-		klog.V(5).Infof("ENABLE_EBPF_CGROUPID: %t", EnabledEBPFCgroupID)
-		klog.V(5).Infof("ENABLE_GPU: %t", EnabledGPU)
-		klog.V(5).Infof("ENABLE_QAT: %t", EnabledQAT)
-		klog.V(5).Infof("ENABLE_PROCESS_METRICS: %t", EnableProcessStats)
-		klog.V(5).Infof("EXPOSE_HW_COUNTER_METRICS: %t", ExposeHardwareCounterMetrics)
-		klog.V(5).Infof("EXPOSE_IRQ_COUNTER_METRICS: %t", ExposeIRQCounterMetrics)
-		klog.V(5).Infof("EXPOSE_ESTIMATED_IDLE_POWER_METRICS: %t. This only impacts when the power is estimated using pre-prained models. Estimated idle power is meaningful only when Kepler is running on bare-metal or with a single virtual machine (VM) on the node.", ExposeIdlePowerMetrics)
-		klog.V(5).Infof("EXPERIMENTAL_BPF_SAMPLE_RATE: %d", BPFSampleRate)
+		klog.V(5).InfoS("Configuration value", "ENABLE_EBPF_CGROUPID", EnabledEBPFCgroupID)
+		klog.V(5).InfoS("Configuration value", "ENABLE_GPU", EnabledGPU)
+		klog.V(5).InfoS("Configuration value", "ENABLE_QAT", EnabledQAT)
+		klog.V(5).InfoS("Configuration value", "ENABLE_PROCESS_METRICS", EnableProcessStats)
+		klog.V(5).InfoS("Configuration value", "EXPOSE_HW_COUNTER_METRICS", ExposeHardwareCounterMetrics)
+		klog.V(5).InfoS("Configuration value", "EXPOSE_CGROUP_METRICS", ExposeCgroupMetrics)
+		klog.V(5).InfoS("Configuration value", "EXPOSE_IRQ_COUNTER_METRICS", ExposeIRQCounterMetrics)
```

Comment:
You can make this all one line:
```suggestion
		klog.V(5).InfoS("Configuration values", "ENABLE_EBPF_CGROUPID", EnabledEBPFCgroupID, "ENABLE_GPU", EnabledGPU, "ENABLE_QAT", EnabledQAT,"ENABLE_PROCESS_METRICS", EnableProcessStats, "EXPOSE_HW_COUNTER_METRICS", ExposeHardwareCounterMetrics, "EXPOSE_CGROUP_METRICS", ExposeCgroupMetrics, "EXPOSE_IRQ_COUNTER_METRICS", ExposeIRQCounterMetrics)
```